### ISSUE TITLE
build not working on redeploy

### ISSUE THREAD

**Issue by jakemiller649** [w201rdada/portfolio-jakemiller649#5]
I am attempting to redeploy in preparation for peer review. build.sh runs into an error and raises the following message. Any guesses?

```
/usr/lib/rstudio-server/bin/pandoc/pandoc +RTS -K512m -RTS _main.utf8.md --to html --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash --output _main.html --smart --email-obfuscation none --standalone --section-divs --table-of-contents --toc-depth 2 --template /usr/local/lib/R/site-library/bookdown/templates/gitbook.html --highlight-style pygments --number-sections --include-in-header /tmp/RtmpkL9djw/rmarkdown-str17d49768975.html --mathjax --filter /usr/lib/rstudio-server/bin/pandoc/pandoc-citeproc 
pandoc-citeproc: "stdin" (line 13, column 2):
unexpected "m"
expecting "c", "C", "p", "P", "s" or "S"
CallStack (from HasCallStack):
  error, called at src/Text/CSL/Input/Bibtex.hs:113:32 in pandoc-citeproc-0.10.4-6rooHIpno7nIk4QQSmQ8di:Text.CSL.Input.Bibtex
pandoc: Error running filter /usr/lib/rstudio-server/bin/pandoc/pandoc-citeproc
Filter returned error status 1
Error: pandoc document conversion failed with error 83
Please delete _main.Rmd after you finish debugging the error.
Execution halted
```

**Comment by jakemiller649** [comment]
It's definitely related to the .bib file, although other people with "error code 83" seem to have different solutions ...

**Comment by jakemiller649** [comment]
SOLVED -- build.sh was pulling in non-standard characters from Zotero in references.bib, which was throwing it for a loop.

**Comment by brooksambrose** [comment]
Glsd you got it sorted. How did you track down the issue? Did you find help online?

**Comment by jakemiller649** [comment]
I searched "pandoc document conversion failed with error 83" until I read enough things that seemed to point to the .bib file (although not everyone who got error 83 was .bib related, and no one else was using Zotero). After a couple failed re-builds I realized that I needed to edit the citation entries in Zotero, not references.bib, since references.bib that was being rebuilt each time through build.sh. 

So I just started deleting characters from the authors and titles of my Zotero list and eventually it worked.
